beautysh/beautysh.py

```diff
@@ -224,18 +270,28 @@ def main(self):
                                  "any in-place beautify.")
         parser.add_argument('--tab', '-t', action='store_true',
                             help="Sets indentation to tabs instead of spaces")
+        parser.add_argument('--force-function-style', '-s', nargs=1, type=int, default=1,
```

Comment:
I think I prefer we call them something, and then document it, than using ints as the values.
Something like, `fnpar`,`fnnopar`,`par` for `function foo()`, `function foo`, and `foo()` respectively.

What do you think?

Reply:
Agreed, fnonly is better!

Reply:
Actually, having `fnonly` seems a bit inconsistent... What about `fnpar`, `fn`, and `par` or `fnpar`, `fnonly`, and `paronly`?

Reply:
yeah, you're right. I just checked in the last set of words (fnpar, fnonly, and paronly)